tests/interpolate/test_interpolate_tools.py

```diff
@@ -12,6 +12,7 @@
 
 from metpy.interpolate import (interpolate_to_grid, remove_nan_observations,
                                remove_observations_below_value,
+                               remove_observations_above_value,
```

Comment:
Can you move this before `remove_observations_below_value` (to keep things alphabetical)?